docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -98,7 +98,7 @@ metrics based on the tracing data.
 * [Sentinel: The Sentinel of Your Microservices](https://github.com/alibaba/Sentinel) 1.7.0 -> 1.8.1
 * Scheduler
   * [Elastic Job](https://github.com/elasticjob/elastic-job) 2.x
-  * [Apache ShardingSphere-Elasticjob](https://github.com/apache/shardingsphere-elasticjob) 3.0.0-alpha
+  * [Apache ShardingSphere-Elasticjob](https://github.com/apache/shardingsphere-elasticjob) 3.x
```

Comment:
Does ShardingSphere confirm this break change? This is pretty new plugin, which was added months ago.

Reply:
This plugin was for 3.0.0-alpha. The ElasticJob 3.0.0 GA released so I think the alpha can be replaced.

Reply:
Or maybe I should rename it to 3.0.0 rather than 3.x.

Reply:
SkyWalking doesn't have preference or requirement. We just ask you to be clear for end users. Nothing more.

This comment is for confirming only for thr future.

Reply:
Understood.